.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: ruby
+script: "bundle exec rspec spec"
```

Comment:
You can make the `spec` task the default in the rake file, and remove this line here.